src/container/use-sticky-header.ts

```diff
@@ -42,7 +45,8 @@ export const useStickyHeader = (
   __stickyHeader?: boolean,
   __stickyOffset?: number,
   __mobileStickyOffset?: number,
-  __disableMobile = true
+  __disableMobile = true,
```

Comment:
It seems like code coverage complains about this line. `__disableMobile` is never `undefined`, so it does not see coverage for the `= true` default. 

Simplest solution is to remove this default, it is not needed